master-firmware/src/robot_helpers/math_helpers.c

```diff
@@ -12,3 +15,24 @@ float angle_delta(float start, float end)
         return res_conj;
     }
 }
+
+bool math_point_is_in_square(poly_t* square, int x, int y)
+{
```

Comment:
It was implemented to circumvent the bug encountered with `is_in_poly`. Now that the bug is fix I should go back to using `is_in_poly`